tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
```

Comment:
It's unrelated, it's just a convenience so that you can write:
`import React from "react";`
instead of
`import * as React from "react";`

If there's no explicit default import, I think it reads better but it is the same result. Ideally this is only used for stubbing in test code, most prod code should use destructured import syntax to allow esbuild to tree-shake the module.